.circleci/config.yml

```diff
@@ -508,6 +540,21 @@ jobs:
       - run: CC=gcc-8 CXX=g++-8 V=1 make -j16 all
       - post-steps
 
+  build-linux-cmake-with-folly-coroutines:
+    machine:
+      image: ubuntu-2004:202111-02
+    resource_class: 2xlarge
+    environment:
+      CC: gcc-10
+      CXX: g++-10
+    steps:
+      - pre-steps
+      - run: sudo apt-get update -y && sudo apt-get install gcc-10 g++-10 libgflags-dev
+      - setup-folly
+      - build-folly
```

Comment:
I guess in other cases the system compiler is ABI compatible with the compiler used with RocksDB, but not in this case? (I would still expect ODR violations between compilers given how much conditional compilation is in folly headers.)

Reply:
In this case, we need g++ 10 for coroutine support.